### PR TITLE
New clown shoes: Moffers

### PR DESCRIPTION
**Pull request by ValuedEmployee**
## About The Pull Request

Adds a new set of clown shoes to the autodrobe under the contraband list that are moth themed. The shoes look like little moth heads on the ends and make moth screaming noises when you take a step. It came to me in a dream and I made this PR in an hour so please be nice to me.

## Why It's Good For The Game

Buying meown shoes has been a staple for certain fun loving players and clowns alike on TG. Adding an extra pair of basically the moth version of them can add extra gimmick shenanigans for clowns to baffle onlookers and elicit a giggle from local assistants. Since autodrobes must be hacked to gain these moffer slippers, with an inventory amount of 1 maximum per autodrobe, they should be somewhat of a rarity that adds a little spice to a round.

https://github.com/tgstation/tgstation/assets/105574010/5099d088-d760-4b2a-823a-03bf9b342da9

Screenshot 2024-03-02 160533
Screenshot 2024-03-02 170927

Most importantly: Moth

## Changelog

:cl:
add: Added new clown shoes "moffers"
add: Added moffers to the contraband list of the autodrobe
/:cl:

### CODE REVIEW

**Comment by dieamond13** [comment]
mothroaches were harmed in the making of these shoes

**Comment by Wallemations** [comment]
Please just make your changelog about the added shoes instead of each individual component that went into adding said shoes

**Comment by ValuedEmployee** [comment]
> Please just make your changelog about the added shoes instead of each individual component that went into adding said shoes

My bad, should be an easier read now.

**Comment by ophaq** [comment]
Finally. Now we just need the clown to be able to make balloon animals.

**Comment by Ticklezcat** [comment]
"make moth screaming noises when you take a step"
"It came to me in a dream"
All the signs of a perfect addition to the game.

File: code/modules/clothing/shoes/clown.dm [hunk 1]
```diff
@@ -53,3 +53,9 @@
 	desc = "The adorable sound they make when you walk will mean making friends is more likely."
 	icon_state = "meown_shoes"
 	squeak_sound = list('sound/effects/footstep/meowstep1.ogg'=1) //mew mew mew mew
+
+/obj/item/clothing/shoes/clown_shoes/moffers
+	name = "moffers"
+	desc = "No moths died in the making of these slippers."
```
Comment (JohnFulpWillard) [inline code/modules/clothing/shoes/clown.dm:59]:
```suggestion
	desc = "The question on whether moths were harmed in the making of these cannot be answered."
```

Comment (JohnFulpWillard) [reply code/modules/clothing/shoes/clown.dm:59]: is moffers supposed to be a play on words for something btw

Comment (ValuedEmployee) [reply code/modules/clothing/shoes/clown.dm:59]: It's a silly way to say moth slippers

**Comment by DATA-xPUNGED** [comment]
Screenshot_2024-03-03-10-53-51-431_com discord-edit

File: code/modules/clothing/shoes/clown.dm (hunk 1 above)
Comment (ValuedEmployee) [reply code/modules/clothing/shoes/clown.dm]: Description changed to include "were harmed" instead of "died"

**Pull request merged**